### DISCUSSION TITLE
Mybinder instance

### DISCUSSION THREAD

**Discussion by sidbannet** [sidbannet/converge_cfd_user#18] [Show and tell]
I created a [mybinder](https://mybinder.org/v2/gh/sidbannet/converge_cfd_user/1d7ab9d64489cf466b5dc98bfc8d33f3f1875228?urlpath=lab%2Ftree%2FAn_open_source_technical_computing_platform_to_analyze_CONVERGE_CFD_results.ipynb) instance for this repo. Check this out.

**Comment by acrlakshman** [comment]
Nice, things are looking fine on binder.
